package/network/services/hostapd/src/src/ap/ubus.c

```diff
@@ -723,6 +725,22 @@ int hostapd_ubus_handle_event(struct hostapd_data *hapd, struct hostapd_ubus_req
 	if (req->frame_info)
 		blobmsg_add_u32(&b, "signal", req->frame_info->ssi_signal);
 	blobmsg_add_u32(&b, "freq", hapd->iface->freq);
+	if (req->elems) {
+		if (req->elems->supp_rates) {
+			supp_rates = blobmsg_open_array(&b, "supp_rates");
+			for (i = 0; req->elems->supp_rates && i < req->elems->supp_rates_len; i++) {
+				blobmsg_add_u8(&b, NULL, req->elems->supp_rates[i]);
```

Comment:
@nbd168: So here I should use e.g. u16?

Reply:
Yes. They take up the same amount of memory anyway (due to alignment)